.nvidia-ci.yml

```diff
@@ -69,7 +67,7 @@ workflow:
 # Download the regctl binary for use in the release steps
 .regctl-setup:
   before_script:
-    - export REGCTL_VERSION=v0.4.5
+    - export REGCTL_VERSION=v0.4.8
```

Comment:
They're at 0.9.2 these days: https://github.com/regclient/regclient/releases

I didn't want to run into breakage; let's see at some point how far we can bump. Maybe cutting edge just works.